src/main/java/it/gov/pagopa/payhub/activities/mapper/exportflow/debtposition/InstallmentExportFlowFileDTOMapper.java

```diff
@@ -0,0 +1,126 @@
+package it.gov.pagopa.payhub.activities.mapper.exportflow.debtposition;
+
+import it.gov.pagopa.payhub.activities.dto.export.debtposition.InstallmentExportFlowFileDTO;
+import it.gov.pagopa.payhub.activities.enums.EntityIdentifierType;
+import it.gov.pagopa.payhub.activities.enums.UniqueIdentifierType;
+import it.gov.pagopa.payhub.activities.util.Utilities;
+import it.gov.pagopa.pu.debtposition.dto.generated.InstallmentPaidViewDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.Person;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+@Lazy
+@Service
+public class InstallmentExportFlowFileDTOMapper {
+
+    public InstallmentExportFlowFileDTO map(Float versionTrack, InstallmentPaidViewDTO installmentPaidViewDTO){
+        switch (versionTrack.toString()){
```

Comment:
due to the usage of the CSV profile utility, we could avoid this:
we could always fill all fields, it will be the CSV profile to handle the writing on the file or not.

just for the data which require other integrations (see RT field) which could introduce an if to explicitly set if the version will require it